src/onemod/config/model_config/rover_config.py

```diff
@@ -49,14 +83,13 @@ class RoverConfig(ModelConfig):
     t_threshold: float = Field(ge=0, default=1.0)
     min_covs: NonNegativeInt | None = None
     max_covs: NonNegativeInt | None = None
-
-    # FIXME: Validate after pipeline settings passed to stage settings
-    # @model_validator(mode="after")
-    # def check_holdouts(self) -> Self:
-    #     """Make sure holdouts present."""
-    #     if self.holdout_columns is None:
-    #         raise ValueError("Holdout columns required for rover stage")
-    #     return self
+    _pipeline_config: Config = Config()
+    _required: set[str] = {
+        "model_type",
+        "observation_column",
+        "weights_column",
+        "holdout_column",
```

Comment:
Is `holdout_column` distinct from `holdout_columns`? I guess I'm misunderstanding the relationship between `_required` items and the class attributes.

Reply:
For context, trying to run `requires_data` tests on the cluster on this branch I get `AttributeError: Missing required config items: ['observation_column', 'weights_column', 'holdout_column']`, although I saw you have updated the test `DummyRoverStage` on this branch.

Reply:
That's a typo! I'm surprised the integration tests for model config passed...

Reply:
Thanks for catching this, my tests weren't written correctly and didn't catch the typo, but now should be fixed.